extension.js

```diff
@@ -112,25 +118,6 @@ function updateSnippetFile(snippetFilename, jsonData) {
             if (err) {
                 throw err;
             }
-
-            var snippetComment = fs.readFileSync(SNIPPET_TEMPLATE_NAME, 'utf8').replace('$(languageId)', path.basename(snippetFilename));
-            var current = fs.readFileSync(snippetFilename, 'utf8');
-            if (!current.startsWith(snippetComment)) {
-                prependFile.sync(snippetFilename, snippetComment, {encoding: 'utf8'});
-            }
-            vscode.workspace.openTextDocument(snippetFilename).then(function(doc) {
-                vscode.window.showTextDocument(doc, vscode.ViewColumn.Two)
-                .then(selectDefaultSnippet);
-            });
```

Comment:
Ahh interesting, yeah makes sense. Maybe we could just make it an explicit action then once the snippet is created through an info bar or something.